app/questionnaire/placeholder_transforms.py

```diff
@@ -183,3 +183,8 @@ def first_non_empty_item(self, items):
             return item
 
         return ""
+
+    def format_suggestions_url(self, url, list_json):
+        if self.language == "en":
+            return url + list_json
+        return ""
```

Comment:
You could achieve the same behaviour here supplying the contatenated url as the arg:

    def format_suggestions_url(self, url):
        return url if self.language in ["en", "cy"] else return ""

Reply:
Done, thanks👍